app/lib/features/events/pages/create_event_page.dart

```diff
@@ -408,9 +409,10 @@ class CreateEventPageConsumerState extends ConsumerState<CreateEventPage> {
 
   // Create event handler
   Future<void> _handleCreateEvent() async {
-    if (!_formKey.currentState!.validate()) return;
+    String? spaceId = ref.read(selectedSpaceIdProvider);
+    spaceId ??= await selectSpace();
```

Comment:
wait ... what sorcery is this? does this ... it means .. `??=` if null run assign to? Wow .... :exploding_head:

Reply:
Reason is that even after showing space selector, user may close it without selecting any space so in that case spaceId can be null and error message will show over there. I have also shown this use-case in screen recording for reference.

Reply:
Check video at 00:34

Reply:
all good. this was a comment of surprise of something new I learnt about the dart language!

Reply:
Ah I see. You always exited about short syntaxes of programming lang.